recipes/dbus/all/conanfile.py

```diff
@@ -0,0 +1,115 @@
+from conans import CMake, ConanFile, tools
+import os
+
+
+class DbusConan(ConanFile):
+    name = "dbus"
+    license = "AFL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.freedesktop.org/wiki/Software/dbus"
+    description = "D-Bus is a simple system for interprocess communication and coordination."
+    topics = ("conan", "dbus")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+            "with_x11": [True, False], 
+            "with_glib": [True, False],
+            "disable_assert": [True, False],
+            "disable_checks": [True, False],
```

Comment:
Double negation (`disable_assert=False`) does not make readable code.
I would change these to `enable_assert` and `enable_checks`.